oct/cli/provision/remote/tests/test_all_in_one.py

```diff
@@ -0,0 +1,73 @@
+# coding=utf-8
+from __future__ import absolute_import, division, print_function
+
+from mock import patch
+import oct.cli.provision.remote.all_in_one as all_in_one_module
+from oct.cli.provision.remote.all_in_one import OperatingSystem, Stage, provision_with_aws
+from oct.tests.unit.playbook_runner_test_case import PlaybookRunnerTestCase, \
+    TestCaseParameters, show_stack_trace, PlaybookRunCallSpecification
+
+if not show_stack_trace:
+    __unittest = True
+
+
+class ProvisionAWSTestCase(PlaybookRunnerTestCase):
+    def setUp(self):
+        PlaybookRunnerTestCase.setUp(self)
+        patches = [
+            patch.object(
+                target=all_in_one_module,
+                attribute='provision_with_aws',
+                new=lambda configuration, _, _, _, _: None,
+                # TODO: Need to patch private_key_path and keypair_name
```

Comment:
@stevekuznetsov any idea how can I initialize the aws configuration correctly for this unit test to work? I need to work around the exceptions thrown in [here](https://github.com/openshift/origin-ci-tool/blob/b58d70e4ba8547fb716da04deb467ce17ccf4345/oct/cli/provision/remote/all_in_one.py#L168-L172).